src/Entities/Hand.ts

```diff
@@ -1,20 +1,60 @@
-import Card from "./Card";
+import Card from './Card'
```

Comment:
You'll notice that it is single quotes and also that the semicolons are removed.
I added a linter to the codebase to use the conventions that I am used to, those changes do not change the functionality of the code.
Basically in JavaScript, you can go either way with quotes and semicolons and whitespace, so this was just an action of cleaning up.